cuckoo_time_translator_algorithms/CMakeLists.txt

```diff
@@ -5,7 +5,11 @@ find_package(catkin_simple REQUIRED)
 catkin_simple()
 
 find_package(Eigen REQUIRED)
+
 find_package(console_bridge REQUIRED)
+if(console_bridge_VERSION GREATER 0.3.0)
+  add_definitions(-DDEFINE_LOGGING_MACROS)
```

Comment:
This macro is passed to all client packages. I'd try to avoid this or if impossible to prefix it with `CUCKOO_TIME_TRANSLATOR` or similar. 
However, in this case we can simply check whether one of them, e.g. `CONSOLE_BRIDGE_logError` is defined to discriminate the two cases directly from the header (assuming the version is hard to test from the preprocessor):

```c++
#ifdef CONSOLE_BRIDGE_logError 
#define  ...

```

Reply:
Didn't know ifdef worked on functions as well, that will actually make changes like this easier in future

Reply:
It only works because it is a (function like) preprocessor macro. For logging you can almost bet on macros because otherwise you cannot capture line number and filename.